witness/src/Dscp/Web/Server.hs

```diff
@@ -41,7 +40,9 @@ buildServantLogConfig
 buildServantLogConfig modifyName = do
     origNameSel <- Log.logName
     logger <- views (lensOf @(Logging IO)) Log._log
-    let origName = fromMaybe mempty $ origNameSel ^? _GivenName
+    let origName = case origNameSel of
+                       GivenName x -> x
```

Comment:
Why indent right so far?

Reply:
4 spaces from `case`, that's the standard practice.